features/git-rename-branch/feature_branch/with_remote_origin/with_tracking_branch/with_open_changes.feature

```diff
@@ -0,0 +1,34 @@
+Feature: git rename-branch: renaming a feature branch with a tracking branch (with open changes)
+
+  As a developer with a poorly named feature branch
+  I want to be able to rename it safely in one easy step
+  So that I can keep my branches well organized and remain productive.
```

Comment:
What do you think about

`So that the names of my branches match what they implement, can I can manage them effectively.`